modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/apps-search-results/ddm-template.ftl

```diff
@@ -178,4 +212,31 @@
 			</#list>
 		</div>
 	</#if>
-</div>
\ No newline at end of file
+</div>
+
+<#if filterCategoriesByUrlParams?has_content>
+	<script>
+		let cards = document.querySelectorAll('a.app-search-results-card');
+		let linkCard = {};
+
+		for (let i = 0; i < cards.length; i++) {
```

Comment:
Do we really need two loops doing the same thing here? Can we just add the others conditions to the same loop? What do you think?